### PR TITLE
fix new address view modal

### PR DESCRIPTION
**Pull request by minhld1709**

### CODE REVIEW

File: app/views/addresses/_new_address_modal_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<%= form_for @address do |f| %>
+  <div class="col-md-12 well">
+    <div class="col-md-12">
+      <%= f.label t "address.latitude" %>
+      <%= f.text_field :lat, class: 'form-control', id: "form-lat" %>
```
Comment (bapboy18) [inline app/views/addresses/_new_address_modal_form.html.erb]: should use "

File: app/views/shared/_modal_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<script type="text/javascript">
```
Comment (cuongnguyen2503) [inline app/views/shared/_modal_form.html.erb]: Write it into js file.

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -2,16 +2,16 @@
...
   <%= csrf_meta_tags %>
 </head>
 <body onLoad="initialize()">
-
   <%= render "layouts/header" %>
   <%= yield %>
-
 </body>
 </html>
+<% unless current_user.nil? %>
```
Comment (cuongnguyen2503) [inline app/views/layouts/application.html.erb]: Should use user_signed_in?

File: app/views/addresses/_new_address_modal_form.html.erb [hunk 4]
```diff
@@ -0,0 +1,31 @@
+<%= form_for @address do |f| %>
+  <div class="col-md-12 well">
+    <div class="col-md-12">
```
Comment (cuongnguyen2503) [inline app/views/addresses/_new_address_modal_form.html.erb]: Should add col-sm.. and col-lg... for support other devices. class="..."

File: app/assets/javascripts/googlemapapi.js [hunk 5]
```diff
@@ -34,7 +34,7 @@ function marker(){
   formlng = document.getElementById("form-lon").value = myLatlng.lng();
 
   var defaultBounds = new google.maps.LatLngBounds(
-    new google.maps.LatLng(20.919650305632143, 106.63326752187459));
+    new google.maps.LatLng(21.01572798012042, 105.7857146859169));
```
Comment (HoangVanTrinh92) [inline app/assets/javascripts/googlemapapi.js]: function LatLngBounds in googlemapapi syntax is 
LatLngBounds((south,west),(north,east))
you should add

File: app/views/addresses/_new_address_modal_form.html.erb [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    </div>
+
+    <div class="col-md-12">
+      <%= f.label t "address.longitude" %><br/>
+      <%= f.text_field :lon, class: "form-control", id: "form-lon" %>
+    </div>
+
+    <div class="col-md-12">
+      <%= f.label t "address.capacity" %><br/>
+      <%= f.text_field :capacity, class: "form-control", id: "form-lon" %>
```
Comment (HoangVanTrinh92) [inline app/views/addresses/_new_address_modal_form.html.erb]: form-lon error

**Pull request merged**